easybuild/easyconfigs/s/ScaFaCoS/ScaFaCoS-1.0.1-foss-2020a.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'ScaFaCoS'
+version = '1.0.1'
+
+homepage = 'http://www.scafacos.de/'
+description = """ScaFaCoS is a library of scalable fast coulomb solvers."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/releases/download/v%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = [
+    '2b125f313795c81b0e87eb920082e91addf94c17444f9486d979e691aaded99b',  # scafacos-1.0.1.tar.gz
+]
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('FFTW', '3.3.8'),
```

Comment:
We usually don't repeat deps that comes from the toolchain do we?
@boegel ??

Reply:
I was just keep consistency with the other easyconfig, no problem to remove.